assembl/static2/js/app/utils/translations.js

```diff
@@ -831,15 +846,29 @@ const Translations = {
         2: "Configure the voting proposals"
       },
       voteWithTokens: "Tokens vote",
-      tokenVoteCheckbox: "Check this box to create a token vote module",
+      voteWithGages: "Gauge(s) vote",
+      tokenVoteCheckbox:
+        "The token vote module allows you to select propositions proportionnaly. Each participant has a certain amount of tokens et will have to spread them on the different propositions",
+      gaugeVoteCheckbox: "You can choose to have one or several gages",
       headerTitle: "Top page Header",
       propositionSectionTitle: "Proposals section",
       instructions: "Instructions",
       voteSessionSuccess: "The vote session is saved with success.",
       exclusive: "Exclusive",
 
       tokenVoteInstructions: "Instructions for the token vote",
-
+      gaugeVoteInstructions: "Instructions for the gauge vote",
+      gaugeNumber: "Gauge number",
+      defineGaugeNumer: "Define gauges number",
+      minValue: "Minimale value",
```

Comment:
No "e" to minimal and maximal (in fact, I think it is minimum and maximum in this case)